pkg/jobmanager/start.go

```diff
@@ -67,6 +67,7 @@ func (jm *JobManager) start(ev *api.Event) *api.EventResponse {
 			}
 			return
 		}
+		_ = jm.emitErrEvent(jobID, EventJobFailed, err)
```

Comment:
this is the actual change

Reply:
Isn't this emitted always now? Even if the job is successful?

Reply:
it's only emitted if it's not a cancelled job: the above `if j.IsCancelled()` has a non-conditional return inside, so either that `if` is executed or this emitErrEvent

Reply:
But in line 79 below it says the job can be non-cancelled and successful.

Reply:
sorry, I misread your comment, yes you're right